dbms/src/DataStreams/MergeSortingBlockInputStream.h

```diff
@@ -131,4 +131,45 @@ class MergeSortingBlockInputStream : public IProfilingBlockInputStream
     bool remerge_is_useful = true;
 };
 
+
+/** Takes stream already sorted by `x` and finishes sorting it by (`x`, `y`).
```

Comment:
Looks like the stream expects that each block in the stream is individually sorted by `description_to_sort` but it isn't reflected in the comment.

Reply:
BTW there is no mention of that in the task. Or is this stream intended to use always in conjunction with `PartialSortingBlockInputStream(description_to_sort)`?